content/tutorial-checks.md

```diff
@@ -0,0 +1,583 @@
+                  "LedgerIndex" : "38E1EF3284A45B090D549EFFB014ACF68927FE0884CDAF01CE3629DF90542D66",
+                  "PreviousFields" : {
+                     "Balance" : "1000000000000",
+                     "Sequence" : 1
+                  },
+                  "PreviousTxnID" : "3E14D859F6B4BE923323EFC94571606455921E65173147A89BC6EDDA4374B294",
+                  "PreviousTxnLgrSeq" : 5
+               }
+            },
+            {
+               "DeletedNode" : {
+                  "FinalFields" : {
+                     "Account" : "rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo",
+                     "Destination" : "rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy",
+                     "DestinationNode" : "0000000000000000",
+                     "DestinationTag" : 1,
+                     "Expiration" : 570113521,
+                     "Flags" : 0,
+                     "InvoiceID" : "6F1DFD1D0FE8A32E40E1F2C05CF1C15545BAB56B617F9C6C2D63A6B704BEF59B",
+                     "OwnerNode" : "0000000000000000",
+                     "PreviousTxnID" : "0FD9F719CDE29E6F6DF752B93EB9AC6FBB493BF989F2CB63B8C0E73A8DCDF61A",
+                     "PreviousTxnLgrSeq" : 8,
+                     "SendMax" : "100000000",
+                     "Sequence" : 4
+                  },
+                  "LedgerEntryType" : "Check",
+                  "LedgerIndex" : "838766BA2B995C00744175F69A1B11E32C3DBC40E64801A4056FCBD657F57334"
+               }
+            },
+            {
+               "ModifiedNode" : {
+                  "FinalFields" : {
+                     "Flags" : 0,
+                     "Owner" : "rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo",
+                     "RootIndex" : "AD136EC2A266027D8F202C97D294BBE32F6FC2AD5501D9853F785FE77AB94C94"
+                  },
+                  "LedgerEntryType" : "DirectoryNode",
+                  "LedgerIndex" : "AD136EC2A266027D8F202C97D294BBE32F6FC2AD5501D9853F785FE77AB94C94"
+               }
+            },
+            {
+               "ModifiedNode" : {
+                  "FinalFields" : {
+                     "Account" : "rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo",
+                     "Balance" : "4999899999952",
+                     "Flags" : 0,
+                     "OwnerCount" : 1,
+                     "Sequence" : 5
+                  },
+                  "LedgerEntryType" : "AccountRoot",
+                  "LedgerIndex" : "D3A1DBAA28717975A9119EC4CBC891BA9A66236C484F03C9911F463AD3B66DE0",
+                  "PreviousFields" : {
+                     "Balance" : "4999999999952",
+                     "OwnerCount" : 2
+                  },
+                  "PreviousTxnID" : "0FD9F719CDE29E6F6DF752B93EB9AC6FBB493BF989F2CB63B8C0E73A8DCDF61A",
+                  "PreviousTxnLgrSeq" : 8
+               }
+            }
+         ],
+         "TransactionIndex" : 0,
+         "TransactionResult" : "tesSUCCESS"
+      },
+      "status" : "success",
+      "validated" : true
+   }
+}
+```
+
```

Comment:
Woah, this is really interesting in a docs-y way. I am woefully unfamiliar with using rippled, and this looks like your workspace to suss out the features so that you can write about it. Just making a side note that this is valuable to me.